mentat/commands.py

```diff
@@ -390,3 +388,18 @@ def argument_names(cls) -> list[str]:
     @classmethod
     def help_message(cls) -> str:
         return "Set a configuration option or omit value to see current value."
+
+
```

Comment:
Something I was thinking about when I was thinking about making a run command is it might be nice to have "put command and its output in llm context" be an option that other commands could use. It's difficult though because commands don't really have output they have `stream.send` statements. And looking through the commands that currently exist none of them make that much sense to put in the llms context (I could imagine the model seeing the user ran `/include file` right before a prompt might change what it focused on?).

I would kind of like to have the commands and their output in `_messages` maybe with some metadata that filters them out before sending to the llm so I could see them with `/conversation`